snippets/System.Text/Rune/csharp/InstantiateRunes.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RuneSamples
+{
+    public class InstantiateRunes
+    {
+        public static void Run()
+        {
+            // <SnippetCodePoint>
+            Rune a = new Rune(0x0061); // LATIN SMALL LETTER A
+            Rune b = new Rune(0x10421); // CRYSTAL BALL
```

Comment:
`U+10421 = DESERET CAPITAL LETTER ER` ([see demo](https://unicode.org/cldr/utility/character.jsp?a=10421&B1=Show)), not "crystal ball"